Gradle build error on windows

I am getting a build error when performing the Gradle build command:

C:\Users\a538209\sagan\sagan-site>c:\users\a538209\sagan\gradlew build -x check
:sagan-client:compileJava UP-TO-DATE
:sagan-client:processResources UP-TO-DATE
:sagan-client:classes UP-TO-DATE
:sagan-client:npmInstall FAILED

FAILURE: Build failed with an exception.
- What went wrong:
  Execution failed for task ':sagan-client:npmInstall'.
  > A problem occurred starting process 'command 'npm''

I have npm installed and it can be called fine from the cmd prompt?

--- a/sagan-client/build.gradle
+++ b/sagan-client/build.gradle
@@ -1,4 +1,5 @@
 import org.gradle.api.tasks.Exec
+import org.apache.tools.ant.taskdefs.condition.Os
 
 apply plugin: 'java'
 
@@ -21,7 +22,11 @@ task npmInstall(type:Exec) {
     logging.captureStandardError LogLevel.LIFECYCLE
     inputs.files "package.json", "bower.json"
     outputs.dir "node_modules"
-    commandLine 'npm', 'install'
+    if(Os.isFamily(Os.FAMILY_WINDOWS)) {
+        commandLine 'cmd', '/c', 'npm', 'install'
+    } else {
+        commandLine 'npm', 'install'
+    }
 }
 
 task npmBuild(dependsOn: npmInstall, type:Exec) {
@@ -30,7 +35,11 @@ task npmBuild(dependsOn: npmInstall, type:Exec) {
     inputs.dir "src"
     inputs.file "gulpfile.js"
     outputs.dir "dist"
-    commandLine 'npm', 'run', 'build'
+    if(Os.isFamily(Os.FAMILY_WINDOWS)) {
+        commandLine 'cmd', '/c', 'npm', 'run', 'build'
+    } else {
+        commandLine 'npm', 'run', 'build'
+    }
 }
 
 jar.dependsOn npmBuild

--- a/sagan-client/package.json
+++ b/sagan-client/package.json
@@ -8,18 +8,17 @@
   "dependencies": {
   },
   "devDependencies": {
-    "bower": "1.3.1",
-    "gulp": "3.5.0",
-    "gulp-util": "2.2.0",
-    "gulp-filter": "0.3.0",
-    "gulp-cram": "0.1.0",
+    "bower": "1.3.2",
+    "gulp": "3.6.0",
+    "gulp-util": "2.2.13",
+    "gulp-filter": "0.4.0",
+    "gulp-cram": "0.2.0",
     "gulp-minify-css": "0.3.0",
-    "gulp-uglify": "0.2.0",
+    "gulp-uglify": "0.2.1",
     "gulp-bower-src": "0.0.1",
-    "gulp-watch": "0.5.0",
-    "when": "2",
+    "when": "3.1.0",
     "jshint": "2.1",
-    "wd": "0.2.10"
+    "wd": "0.2.17"
   },
   "directories": {
     "test": "test"

